src/sentry/web/frontend/debug/mail.py

```diff
@@ -456,7 +464,29 @@ def build_usage_summary():
             int(random.weibullvariate(5, 1) * random.paretovariate(0.2)),
         )
 
-    def build_report():
+    def build_calendar_data(project):
```

Comment:
are we able to add a test which at least runs over this code?

i kind of figured the high level integration tests would, but codecov showing it red